utils/opencas.py

```diff
@@ -154,9 +154,8 @@ def io_class_load_config(cls, cache_id, ioclass_file):
 
     @classmethod
     def start_upgrade(cls):
-        cmd = [cls.casadm_path,
-                    '--script',
-                    '--upgrade-in-flight']
+        cmd = [cls.casadm_path, '--script', '--upgrade-in-flight']
+
         return cls.run_cmd(cmd)
```

Comment:
Shouldn't You merge this change with previous commit and move it there?

Reply:
Done